app/views/supplies/index.html.haml

```diff
@@ -1,12 +1,26 @@
-%table.table.table--bordered.table--striped.table--linked
+= link_to "Add New Supply", new_supply_path, class: "btn btn-default"
+
+%table.table.table-striped.table-linked.table-condensed
   %thead
     %tr
       %th Name
       %th Short Code
+      %th
   %tbody
     - @supplies.each do |supply|
-      %tr
+      %tr{ class: supply.orderable? ? "" : "danger" }
         %td
-          %span.cell--primary= supply.name
+          -if supply.orderable
+            = supply.name
+          -else
+            %s= supply.name
         %td
           %span= supply.shortcode
+        %td
+          .toggle_orderable_button
+            - if supply.orderable?
+              = button_to toggle_orderable_supply_path(supply), method: :patch, class: "btn btn-danger" do
+                = icon :remove
+            - else
+              = button_to toggle_orderable_supply_path(supply), method: :patch, class: "btn btn-default" do
+                = icon :ok
```

Comment:
Probably not for this PR, but I recommend pulling these bits of logic out in to helper methods to clean up the logic in these views and move it to a method that can be easily tested and reasoned about.